wetmap/src/components/newModals/commentsModal/view.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { SetStateAction } from 'react';
+import screenData from '../screenData.json';
+import style from './style.module.scss';
+import Icon from '../../../icons/Icon';
+import ButtonIcon from '../../reusables/buttonIcon';
+import TextInputField from '../textInput';
+
+type CommentsModalViewProps = {
+  handleCommentInsert:  () => void
+  setCommentContent:    (value: SetStateAction<string>) => void
+  setReplyTo: (value: SetStateAction<(number|string)[] | null>) => void
+  commentContent: string
+  replyTo: (number|string)[] | null
+  getCommentListView: (commentId: number | null, level?: number) => React.JSX.Element
+  onClose?:                     () => void
+};
+
+export default function CommentsModalView(props: CommentsModalViewProps) {
+  return (
+    <>
+      <div>
+      <ButtonIcon
+          icon={<Icon name="chevron-left" style={{ scale: '2' }} />}
+          className="btn-lg mt-4 text-gray"
+          onClick={() => {
+            if (props.onClose) {
+              props.onClose();
+            }
+          }}
+        />
+      <h1 className="mb-0 pb-4">{screenData.CommentsModal.header}</h1>
+      </div>
+
+      <div className={style.middleContainer}>
+        {' '}
+        {props.getCommentListView(null)}
+      </div>
+
+      <div className={style.commentEntryContainer}>
+        {props.replyTo
+          ? (
+              <div className={style.replyLine}>
+                <p className={style.userText}>
+                  @
+                  {props.replyTo[0]}
+                </p>
+                <Icon
+                name="close"
+                fill="darkgrey"
+                width="20"
+                onClick={() => props.setReplyTo(null)}
+                 />
+              </div>
+            )
+          : null}
+        <div className={style.replyBox}>
+            <TextInputField
```

Comment:
Please let's use `reusables\textInput` instead

Reply:
Can you add that in? I am not sure how you have set that one up.
Also,  I think after we do the merge of the next round of PRs, we should also go through and get rid of the now unused components, that we can like old modals and older reuasbles.

Reply:
Ok, input Switched to the newer one, however the older one cannot yet be deprecated as the search tool is still using it at this time